### PR TITLE
Add unit tests for com.github.unclecatmyself.common.utils.RedisUtil

### PR DESCRIPTION
**Pull request by Braavos6**
I've analysed your codebase and noticed that `com.github.unclecatmyself.common.utils.RedisUtil` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important.

### CODE REVIEW

**Review by freddytuxworth — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -92,7 +92,13 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>${commons.lang3.version}</version>
 		</dependency>
-	</dependencies>
```
Comment (petersear) [inline pom.xml]: Indent looks wrong.
</dependencies> shouldn't have moved and the added indent should line up with others at the same level.

File: src/test/java/com/github/unclecatmyself/common/utils/RedisUtilTests.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        Assert.assertEquals("", RedisUtil.convertMD5(""));
+        Assert.assertEquals("EFG", RedisUtil.convertMD5("123"));
+        Assert.assertEquals("\u0012\u001B\u001B", RedisUtil.convertMD5("foo"));
+        Assert.assertEquals("PQ*R^", RedisUtil.convertMD5("$%^&*"));
+    }
+
+    @Test
+    public void testConvertMD52() {
+        Assert.assertEquals("\u001A\u0001\u0018\u0018R\u001A\u0001\u0018" +
+                        "\u0018Q=\u001A7\u001C\u0015\u0000",
```
Comment (petersear) [inline src/test/java/com/github/unclecatmyself/common/utils/RedisUtilTests.java]: I think the indents of continuation lines should line up (possibly just my preference)

**Review by petersear — CHANGES_REQUESTED** [review]

**Review by petersear — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (Braavos6) [reply pom.xml]: Done

File: src/test/java/com/github/unclecatmyself/common/utils/RedisUtilTests.java (hunk 2 above)
Comment (Braavos6) [reply src/test/java/com/github/unclecatmyself/common/utils/RedisUtilTests.java]: Done

File: pom.xml [hunk 3]
```diff
@@ -92,6 +92,12 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>${commons.lang3.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
```
Comment (paulkennethkent) [inline pom.xml:108]: Speculating as I normally do but as they didn't have tests or JUnit previously then are you sure these tests run successfully outside the IDE?

**Review by paulkennethkent — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (Braavos6) [reply pom.xml:108]: Done

**Pull request closed without merging**